content/docs/azure/authentication.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Restricting Access & Authentication "
+description = "Restrict access of your deployment to specified IP addresses"
+weight = 100
++++
+
+This section shows how to restrict access to only certain IP addresses for your LoadBalancer Service. At a later date, it will also include formal authentication through Azure. This method is not the most ideal way to secure your Kubernetes cluster, as it requires that you access the service from the same IP address every time. This process was adapted from [this guide](https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service).
+
+
+When using a Service with `spec.type: LoadBalancer`, you can specify the IP ranges that are allowed to access the load balancer by using `spec.loadBalancerSourceRanges`. This is currently supported on all major cloud providers. 
+
+### Editing the LoadBalancer service
+Use the `kubectl edit svc <loadbalancer-name> -n kubeflow` to add your source ranges. Pressing `i` will allow you to insert text, and `esc` followed by `:wq` will allow you to write and quit out of the text editor.
```

Comment:
This looks like the text editor is VI or something similar. Does the `kubectl edit` utility always invoke the same editor, no matter which OS the person is using, and no matter what they've set as their default editor? (I haven't tried it, so I'm asking from a user's point of view. I've set my default editor on my Mac and Linux machines to Pico. I don't use Windows for Kubeflow.)